Packs/Confluence/Integrations/Confluence/Confluence.yml

```diff
@@ -29,6 +29,7 @@ configuration:
 script:
   script: ''
   type: python
+  subtype: python3
```

Comment:
I think that since there is no docker, this may actually be python2 by default.
need to check this.